cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -244,4 +253,16 @@ function up() {
     # [1] https://github.com/kubevirt/kubevirtci/issues/906
     # [2] https://github.com/k8snetworkplumbingwg/multus-cni/issues/982
     copy_istio_cni_conf_files
+
+    deploy_kwok
+    until wait_for_kwok_ready; do
```

Comment:
Would it be possible to move this wait to line 247 with the other waits?

Reply:
done

Reply:
Hey @alaypatel07  - I don't see this included with the other waits for cluster compnents? 

https://github.com/kubevirt/kubevirtci/pull/1178/files#diff-592e26bb575a60fc920cb2ab9f9cb0fd10609ebd36b19c35ddd1d57ba0fc0cd3R274

Reply:
sorry I mis-understood as moving the function signature. Will fix and move it in a few mins.